src/CommunityToolkit/Xamarin.CommunityToolkit/Views/DrawingView/Extensions/Extensions.shared.cs

```diff
@@ -0,0 +1,76 @@
+using System.Collections.ObjectModel;
+using System.Linq;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.UI.Views
+{
+	/// <summary>
+	/// Extension methods to support <see cref="DrawingView"/>
+	/// </summary>
+	public static class Extensions
+	{
+		/// <summary>
+		/// Get smoothed path.
+		/// </summary>
+		public static ObservableCollection<Point> SmoothedPathWithGranularity(this ObservableCollection<Point> currentPoints,
```

Comment:
extracted from iOS and applied smoothed path to all platforms

Reply:
This will change something in the final result for the user? If so, we can't do this here

Reply:
Actually it was a bug if line smooth enabled.
This functionality worked only on iOS. But expected to work on all platforms.
You can compare the current version with the main branch.